buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -45,7 +45,12 @@ internal class YamlWriter {
   internal fun writeFlankProperties(config: FladleConfig): String = buildString {
     appendln("flank:")
 
-    appendProperty(config.testShards, name = "max-test-shards")
+    // To preserve backward compatibility. To be removed once testShards is deleted
```

Comment:
We should fail the build if both properties are present as it indicates a user error and we can't be sure which one should take priority.

Reply:
Sure, thanks for the input 👍